lib/Parse/CMakeLists.txt

```diff
@@ -6,6 +6,9 @@ else()
   set(SWIFT_GYB_FLAGS --line-directive "\'#line" "%(line)d" "\"%(file)s\"\'")
 endif()
 
+add_subdirectory(${SWIFT_SOURCE_DIR}/../swift-syntax ${CMAKE_CURRENT_BINARY_DIR}/swift-syntax)
```

Comment:
We already have `SWIFT_PATH_TO_EARLYSWIFTSYNTAX_BUILD_DIR` for this, which uses exported targets. Please build on top of that infrastructure.